pkg/executables/govc.go

```diff
@@ -302,7 +309,29 @@ func (g *Govc) deployTemplate(ctx context.Context, library, templateName, deploy
 		return fmt.Errorf("failed writing deploy options file to disk: %v", err)
 	}
 
-	params := []string{
+	params := []string{"folder.info", deployFolder}
+	err = g.retrier.Retry(func() error {
+		_, err = g.executable.ExecuteWithEnv(ctx, envMap, params...)
+		if err != nil {
```

Comment:
So this would retry the folder.info call if folder.create fails right? Can we move them to separate retries, where we retry the folder.create outside of the folder.info retry?

Reply:
Oh good catch, yes, will move.

Reply:
Let's check for "not found" string here instead, it will clean up the logical flow